rego/rego_test.go

```diff
@@ -1750,6 +1750,13 @@ func TestEvalWithInterQueryCache(t *testing.T) {
 	}
 }
 
+func TestStrictBuiltinErrors(t *testing.T) {
+	_, err := New(Query("1/0"), StrictBuiltinErrors(true)).Eval(context.Background())
+	if err == nil || err.Error() != "1/0: eval_builtin_error: div: divide by zero" {
```

Comment:
Unrelated: an error refactoring could be useful -- allowing the caller to differentiate stuff like this based on types, not strings.

Reply:
Fixed.